src/workerd/server/server-test.c++

```diff
@@ -632,7 +632,7 @@ KJ_TEST("Server: serve basic modular Worker") {
       )
     ]
   ))"_kj));
-
+  KJ_EXPECT_LOG(ERROR, "Autogates not initialised, check for builtin-wasm-modules will have no effect");
```

Comment:
Wouldn't it be better to make the test hoise initialize the autogates?

Reply:
yeah, though I'd rather leave this as TODO for a separate PR.